dusty/cli/__init__.py

```diff
@@ -125,7 +126,10 @@ def main():
     args = docopt(__doc__, options_first=True)
     command, command_args = args['<command>'], args['<args>']
     if command is None:
-        print __doc__.strip()
+        if '-v' in sys.argv:
+            print 'Dusty version {}'.format(constants.VERSION)
```

Comment:
I was thinking that it might be possible to have -v in the argv's in some other context. Do you think I should move it to above?